.vscode/cspell.json

```diff
@@ -839,8 +839,12 @@
     {
       "filename": "sdk/servicebus/azure-messaging-servicebus/**",
       "words": [
+        "anu",
         "Conniey",
+        "liudmila",
+        "Milli",
         "qpid",
+        "roller",
```

Comment:
surprised that this being picked up as a word that doesn't exist.

Reply:
I just ran cspell locally without this entry and it passed. I've removed word 'roller'. thanks!